index.d.ts

```diff
@@ -0,0 +1,64 @@
+import * as redux from 'redux';
+
+export class Store<T> {
+	/**
```

Comment:
Seems like you're using tabs (or the spacing on your indents are off). Could you modify the lint scripts in `package.json` to include `*.ts` as well? Also any modifications to `.eslintrc` to get that working would be much appreciated! :-)